snap/snapcraft.yaml

```diff
@@ -12,7 +12,7 @@ confinement: strict
 
 package-repositories:
   - type: apt
-    cloud: bobcat
+    cloud: caracal
```

Comment:
Not sure we want to build for Squid by default before having a stable upstream release.
In any case we should cut a reef branch before switching master

Reply:
I think these are independent. As soon as we switch the reef track to a stable branch we can and should also build/publish the squid snap (without a stable risk level maybe).